charts/oes/charts/spinnaker/templates/configmap/halyard-init-script.yaml

```diff
@@ -44,16 +44,30 @@ data:
     # additionalInitScript
     {{ tpl .Values.halyard.additionalInitScript $ | indent 4 }}
     {{- end }}
-    {{- else }}
+    
+    {{- else if contains "git" .Values.gitopsHalyard.repotype }}
     #!/bin/bash -x
+    echo "+++++++++++running git commands+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++"
     git clone $GIT_CLONE_PARAM /tmp/spinnaker/.hal
     GIT_USER=`echo $GIT_USER | sed 's/ *$//g'`
```

Comment:
Line 52 and 53 will also go under {{- if .Values.gitopsHalyard.enabled }}

Reply:
done